pages/stats/PKPStatsHandler.inc.php

```diff
@@ -133,7 +133,7 @@ public function editorial($args, $request) {
 		$activeByStage = [];
 		foreach (Application::get()->getApplicationStages() as $stageId) {
 			$activeByStage[] = [
-				'name' => Application::get()->getWorkflowStageName($stageId),
+				'name' => __(Application::get()->getWorkflowStageName($stageId)),
 				'count' => Services::get('editorialStats')->countActiveByStages($stageId, $args),
 				'color' => Application::get()->getWorkflowStageColor($stageId),
```

Comment:
Maybe it's better to move this `getWorkflowStageColor` to the Vue component?

Reply:
I'd be comfortable with that. The colors should be defined as less variables but since this needs to go into the data passed down to Chart.js it probably isn't possible to reuse them.